repos/system_upgrade/el7toel8/actors/firewalldisable/actor.py

```diff
@@ -85,10 +81,8 @@ def process(self):
                 return
         else:
             self.log.info("Interrupting: There was nothing to consume regarding the Firewall decision.")
-            self.produce(
-                Inhibitor(
-                    summary='No message to consume',
-                    details='No decision message to consume.',
-                    solutions=None
-                    ))
+            report_generic(
+                severity='low',
+                title='No message to consume',
+                summary='No decision message to consume.')
```

Comment:
Inhibitor changed to nothing?  This is fishy; - yep, previous code looks fishy too.

Reply:
Forgot the flag. Will fix it.

Reply:
Forgot the flag. Will fix it.